src/YesSql.Core/Utils/HashHelper.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace YesSql.Utils
+{
+    public static class HashHelper
+    {
+        // Use of numeric characters is supported as long as the name does not start with a number.
```

Comment:
Where is this enforced in the code?

Reply:
I think it's important to write a comment somewhere that we use a deterministic hash instead of a random number so we can find the name of the key later on. I was asking myself why it was not a random number, smaller, while looking at the PR.

Reply:
Added comments to the `ISqlDialect` methods that it is deterministic.

Reply:
> Where is this enforced in the code?

Changed the hash helper so it has to take a prefix. 

I didn't validate that the incoming prefix is a char, not numeric, that seems the responsibility of the dialect.